src/main/java/com/yanirta/ImageTester.java

```diff
@@ -163,13 +166,13 @@ private static void runTestWithBatchMapper(final Logger logger, final CommandLin
                 currentConfiguration.forcedName = currentBatch.testName;
                 currentConfiguration.sequenceName = cmd.getOptionValue("sq", null);
                 currentConfiguration.legacyFileOrder = cmd.hasOption("lo");
-                currentConfiguration.setViewport(cmd.getOptionValue("vs", null));
-                currentConfiguration.setMatchSize(cmd.getOptionValue("ms", null));
-                currentConfiguration.setBatchInfo(currentBatch.batchName, cmd.hasOption("nc"));
+                currentConfiguration.setViewport(currentBatch.viewport.length() == 0 ? null: currentBatch.viewport);
+                currentConfiguration.setMatchSize(currentBatch.matchsize.length() == 0 ? null: currentBatch.matchsize);
```

Comment:
Let's replace `currentBatch.viewport.length() == 0` with `currentBatch.viewport.isBlank()`. 
That will cover cases where someone puts spaces between the pipes, but will still work for an empty string.